src/js/darkwire.js

```diff
@@ -105,46 +101,78 @@ export default class Darkwire {
     return importKeysPromises;
   }
 
+  createUser(resolve, reject, username) {
+    Promise.all([
```

Comment:
I think this is leftover but Promise.all isn't needed for just one promise so we can remove it.